posixlib/src/main/scala/scala/scalanative/posix/spawn.scala

```diff
@@ -63,7 +63,7 @@ object spawn {
       newfiledes: CInt
   ): CInt = extern
 
-  def posix_spawn_file_actions_open(
+  def posix_spawn_file_actions_addopen(
```

Comment:
I concur, this was a mistake in the initial implementation.
Good catch.

I noticed in checking this that  Open Group Issue 8 added a few items. I'll probably add them after this PR merges.